gql/filter.go

```diff
@@ -0,0 +1,173 @@
+package gql
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/GannettDigital/graphql"
+	"github.com/GannettDigital/graphql/language/ast"
+	"github.com/GannettDigital/graphql/language/kinds"
+)
+
+var (
+	// ListOperations is the set of list filters operations available for use by ResolveListField.
+	// The string is the operation, ie '==' which matches a particular NewListOperation function.
+	// The default implementations are pre-populated and additional operations can be added before running the
+	// ObjectBuilder.
+	ListOperations = map[string]NewListOperation{
+		"==":          NewEqualComparator,
+		"!=":          NewNotEqualComparator,
+		">":           NewIntegerComparator(">"),
+		">=":          NewIntegerComparator(">="),
+		"<":           NewIntegerComparator("<"),
+		"<=":          NewIntegerComparator("<="),
+		"limitLength": NewLimitLengthComparator,
+		"IN":          NewInComparator,
+		"NOT IN":      NewNotInComparator,
+		"in":          NewInComparator,
```

Comment:
we should be consistent and only support 1 case for the operations. IMO, they should be `LIMIT, IN, NOT IN`

Reply:
I admit I don't like limitLength now that I see it next to the others. I could go either way with multiple case, so unless there are other opinions I'll stick to these recommendations.

Reply:
Why not case insensitive?  SQL is.   And I agree with Devo about `limit` rather than `limitLength`.

Reply:
I could change it to make all operations case insensitive, though in retrospect probably best to do this for all operations rather than just selectively if we decide that is best.